deepfence_frontend/apps/dashboard/src/features/postures/pages/PostureCloudDetailModal.tsx

```diff
@@ -68,8 +69,19 @@ const Header = () => {
   );
 };
 
+const UPPERCASE_WORDS = ['id'];
```

Comment:
this we need to do in all node details modals.

Reply:
Yes we need in all details modals